src/store/workflows.module.js

```diff
@@ -35,7 +35,7 @@ export function merge (data, update) {
   }
 }
 
-const state = {
+const state = () => ({
```

Comment:
Excuse my JS ignorance, but why this change?

Reply:
I assume this is similar to the way component `data` is provided as a function.

This is providing the initial data for the store. The way it was done before is perfectly legit due to the way we are using it, however, if we tried to use multiple workflow stores on the app it would break. Using a function avoids this problem and is probably documented as best practice.